pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1548,9 +1548,9 @@ public void checkTopicNsOwnership(final String topic) throws BrokerServiceExcept
         List<CompletableFuture<Void>> closeFutures = Lists.newArrayList();
         topics.forEach((name, topicFuture) -> {
             TopicName topicName = TopicName.get(name);
-            if (serviceUnit.includes(topicName)) {
+            if (getTopicReference(name).isPresent() && serviceUnit.includes(topicName)) {
```

Comment:
in the linked PR there was a comment to move `getTopicReference(name).isPresent()` as last operator

probably it is the case to fix it here

Reply:
@Technoboy- Please check the comment.

Reply:
Yes, I see, thanks.
@eolivelli @codelipenghui .